composer.json

```diff
@@ -94,16 +94,16 @@
 		"process-timeout": 0
 	},
 	"scripts": {
-		"test": "php ../../tests/phpunit/phpunit.php -c phpunit.xml.dist",
-		"phpunit": "php ../../tests/phpunit/phpunit.php -c phpunit.xml.dist",
-		"phpdbg": "phpdbg -qrr ../../tests/phpunit/phpunit.php -c phpunit.xml.dist",
+		"test": "php ${MW_INSTALL_PATH}/tests/phpunit/phpunit.php -c phpunit.xml.dist",
```

Comment:
@kghbln @JeroenDeDauw Is this ok?  I was having trouble with symlinks and this made it work.

Reply:
Seems reasonable enough to me. Have not tried it myself. Nowadays I use make like seen here https://github.com/ProfessionalWiki/ExternalContent/blob/master/Makefile